src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &-img {
+    height: 134px;
+    width: 160px;
+    margin: 16px auto 40px;
+    object-fit: fill;
+  }
+
+  &-content {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $card-main-color;
+  }
+
+  &-description {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &-code {
+    color: $card-secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &-rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &-price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+    &-lable {
+      font-size: 12px;
+      color: $card-secondary-color;
+    }
+
+    &-cost {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &-button {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &-link {
+    height: $card-button-height;
+    line-height: $card-button-height;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-align: center;
+    color: $card-button-hover-color;
+    background-color: $card-button-color;
+    border-radius: 5px;
+  }
+
+  &-link:hover {
+    color: $card-button-color;
+    box-sizing: border-box;
+    background-color: $card-button-hover-color;
+    border: 1px solid $card-button-color;
+  }
```

The solution is not optimal - I would suggest using the following: 

```suggestion
  &-link {
    line-height: $card-button-height;
    text-decoration: none;
    font-size: 14px;
    font-weight: 700;
    text-transform: uppercase;
    text-align: center;
    color: $card-button-hover-color;
    background-color: $card-button-color;
    border-radius: 5px;
    border: 1px solid $card-button-color;
  }

  &-link:hover {
    color: $card-button-color;
    background-color: $card-button-hover-color;
  }
```

0. Make sure that button height is correct. Don't use height. You may keep line-height, use font size + padding - that is the recommended way.

1. Set the border for the link and never change it. 
2. Changing only background and font color does all the job.
3. Avoid changing box-sizing on hover. I would suggest just avoid using it at all.
4. General rule - change as little styles as possible.